config/config.go

```diff
@@ -218,6 +218,8 @@ type Config struct {
 
 	PeersRatingConfig   PeersRatingConfig
 	PoolsCleanersConfig PoolsCleanersConfig
+
+	FireHoseConfig FireHoseConfig
```

Comment:
This is not ok.

Reply:
Closed, since we decided to move the firehose custom logic into a new repo: https://github.com/multiversx/mx-chain-ws-connector-firehose-go